### DISCUSSION TITLE
Tmux-navigator | working only if I launch nvim specifying a file

### DISCUSSION THREAD

**Discussion by MatthieuTricoire** [LazyVim/LazyVim#1488] [Q&A]
Hi guys,

I have a problem with christoomey's tmux-navigator that I can't solve.

The plugin is correctly installed on the lazyvim side and it is also correctly installed on the tmux side.

I open a tmux session, in which I add a panel.
I navigate between the two without any problem.

In the top panel, I open neovim with the following command: nvim ~/.config/nvim

I can navigate between the two panels without any problem.

I close neovim, go to a project folder and launch neovim with the following command: nvim.

I can't leave nvim to go to the second tmux panel.

Can you think of anything I've done wrong or am missing?

**Comment by dpetka2001** [comment]
Not able to reproduce. It still works for me even without arguments. You can take a look at the [commit](https://github.com/dpetka2001/dotfiles/commit/9cd061381cf15e7444add6b4ab6e5064de04c41f), in which I added the necessary support for my configuration.
